repository/repository.go

```diff
@@ -25,12 +26,39 @@ const (
 type Repository struct {
 	userRepoPath string
 	forkRepoPath string
+	basePath     string // defaults to ~/.config/container-use if empty
+}
+
+// getBasePath returns the base path for container-use data, using the default if not set
+func (r *Repository) getBasePath() string {
```

Comment:
nit: This logic is already handled by `Open()` (e.g. setting `cuGlobalConfigPath` when calling `Open()`).

I think it would be simpler to avoid duplication of this logic, always assume `r.basePath` is set, and get rid of `getBasePath` altogether